### PR TITLE
First draft of July CSSWG updates

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description

Another CSSWG update that is probably longer than needed…

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@jgerigmeyer @SondraE @stacyk another CSSWG update, ready to review. I set the publishing date to next tuesday - but we could move it up to this thursday if it's ready.

File: content/blog/2024/csswg-08.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
+---
+title: Partial Feature Queries, Relaxed Layout Containment, and more
+sub: CSS working group updates from July
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:3]:
```suggestion
sub: CSS Working Group updates from July
```

File: content/blog/2024/csswg-08.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+Wouldn't it be nice
+to have a way for users
+to change their preference site-by-site,
+and have browsers remember their preference?
+That's what the proposed
+[Web Preferences API](https://github.com/WICG/web-preferences-api)
+would do.
+This proposal gives us access
+to update a domain-specific setting
+a any `prefers-*` query, for any user,
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:146]:
```suggestion
for any `prefers-*` query, for any user,
```

File: content/blog/2024/csswg-08.md [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+and have the browser remember that setting
+when they return.
+
+That proposal has been underway for some time,
+but last month it was
+[adoped by the CSS Working Group](https://github.com/w3c/csswg-drafts/issues/6517#issuecomment-2220975121),
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:152]:
```suggestion
[adopted by the CSS Working Group](https://github.com/w3c/csswg-drafts/issues/6517#issuecomment-2220975121),
```

File: content/blog/2024/csswg-08.md [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+
+Part of the reason
+for moving Container Queries
+out of the Containment spec
+is a drive to relax the relationship
+between these features.
+Containment is essential for
+making size queries possible --
+but the existing breakdown of containment types
+defined for the `contain` property,
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:166]:
```suggestion
defined for the `contain` property
```

File: content/blog/2024/csswg-08.md [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+we should remove the ones that aren't necessary.
+
+Back in March we resolved that
+[anchor names are not blocked by `style` or `layout` containment](https://github.com/w3c/csswg-drafts/issues/10040#issuecomment-2009840674).
+I immediately opened an issue
+pointing out that this situation is
+[not specific to anchor positioning](https://github.com/w3c/csswg-drafts/issues/10102),
+and we should consider allowing all positioned elements
+to escape `layout` containment --
+and therefor escape containers.
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:206]:
```suggestion
and therefore escape containers.
```

File: content/blog/2024/csswg-08.md [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+The work paid off.
+
+In this case,
+we can't fully _have it both ways_.
+Masonry will either be part of grid layout,
+or it will have to be separate.
+But in many ways,
+we're taking the same approach:
+Chrome is opening issues
+to show where there masonry might
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:381]:
```suggestion
to show where their masonry might
```

File: content/blog/2024/csswg-08.md [hunk 7]
```diff
@@ -0,0 +1,410 @@
...
+and WebKit is looking for ways
+to minimize those conflicts.
+Depending how those conflicts are resolved,
+we'll have a much better sense
+if the masonry-in-grid approach is viable or not.
+
+And then we will (hopefully)
+not be debating how the feature _works_,
+but only how it is 'spelled' in CSS syntax.
+Maybe kicking that can down the road,
```
Comment (jgerigmeyer) [inline content/blog/2024/csswg-08.md:392]:
```suggestion
Maybe kicking that can down the road
```

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
- The M in _more_ in the title should be uppercase.
- "...month, The CSS Working..." in this sentence in the blurb, I believe the T in _The_ should be lowercase
- "What’s up with CSS Masonry?" To maintain internal article capitalization consistency, the U in _up_ should be uppercase.
- "Also keep an eye on…" The K E and O should be uppercase.

**Review by stacyk — APPROVED** [review]
Looks great! The last CSSWG post we also included Container Queries as a tag and I wonder if we want to include it on this one as well?

**Pull request merged**